lib/Dialect/Torch/Transforms/DecomposeComplexOps.cpp

```diff
@@ -6656,6 +6801,7 @@ class DecomposeComplexOpsPass
         DecomposeAtenAddCLikeOp<AtenAddcmulOp, AtenMulTensorOp>>(patterns);
     addPatternIfTargetOpIsIllegal<
         DecomposeAtenAddCLikeOp<AtenAddcdivOp, AtenDivTensorOp>>(patterns);
+    addPatternIfTargetOpIsIllegal<DecomposeAtenInstanceNormOp>(patterns);
```

Comment:
Can you also add the op here: 

https://github.com/llvm/torch-mlir/blob/d6e1d836ca8d49da25ad5e2f10d6816bfbb6ba2f/lib/Dialect/Torch/Transforms/LowerToBackendContract.cpp#L367